ui/components/ui/review-spending-cap/review-spending-cap.js

```diff
@@ -64,7 +65,13 @@ export default function ReviewSpendingCap({
                   color={COLORS.TEXT_ALTERNATIVE}
                   className="review-spending-cap__heading-title__tooltip"
                 >
-                  {tokenValue > currentTokenBalance &&
+                  {conversionGreaterThan(
```

Comment:
Same as my comment above, store the result of this `conversionGreaterThan` call in a variable and use it through the component